cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/AwsCredentialConnector.java

```diff
@@ -125,19 +125,24 @@ public CloudCredentialStatus delete(AuthenticatedContext auth) {
     }
 
     @Override
-    public CredentialPrerequisitesResponse getPrerequisites(CloudContext cloudContext, String externalId, String deploymentAddress, CredentialType type) {
+    public CredentialPrerequisitesResponse getPrerequisites(CloudContext cloudContext, String externalId,
+        String auditExternalId, String deploymentAddress, CredentialType type) {
```

Comment:
Why do you create a separate external id? It looks like you don't use both together

Reply:
the actual id depends on the credential type